querydsl-docs/src/main/docbook/en-US/content/general/codegen.xml

```diff
@@ -196,7 +196,14 @@ import com.mysema.query.annotations.Config;
             <entry>querydsl.excludedClasses</entry>
             <entry>comma separated list of class names to be excluded from code generation (default: none)</entry>
           </row>
-
+          <row>
+            <entry>querydsl.useFields</entry>
+            <entry>set whether fields are used as metadata source (default: true)</entry>
```

Comment:
Yes, by default we process everything.
However with some submodules we inspect the presence of relevant annotations first.